src/cb/token.cr

```
@@ -41,6 +41,11 @@ struct CB::Token
     fetch? host
   end
 
+  def self.delete(host)
+    token = fetch host
+    token.delete unless token.nil?
```

I kind of feel like `Token#delete` and also `Creds#delete` should perhaps be more fault tolerant in the long run. However, I figured since it's not `nil` then it 'should' exist and should be safe. That doesn't mean I don't still wish it to be more fault tolerant, but I also don't want to explicitly silence any exceptions... not yet at least without good reason.